tests/acceptance/shared/util/util.go

```diff
@@ -0,0 +1,199 @@
+package util
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"golang.org/x/crypto/ssh"
+)
+
+// RunCommandHost executes commands on the host
+// return the output of it and an error if any
+func RunCommandHost(cmd ...string) (string, error) {
+	var output bytes.Buffer
+	for _, cmd := range cmd {
+		c := exec.Command("bash", "-c", cmd)
+
+		stdoutPipe, err := c.StdoutPipe()
+		if err != nil {
+			return "", err
+		}
+		stderrPipe, err := c.StderrPipe()
+		if err != nil {
+			return "", err
+		}
+
+		err = c.Start()
+		if err != nil {
+			return "", err
+		}
+
+		_, err = io.Copy(&output, stdoutPipe)
+		if err != nil {
+			return "", err
+		}
+
+		_, err = io.Copy(&output, stderrPipe)
+		if err != nil {
+			return "", err
+		}
+
+		err = c.Wait()
+		if err != nil {
+			return output.String(), fmt.Errorf("error executing command: %s: %w", cmd, err)
+		}
+	}
+
+	return output.String(), nil
+}
+
+// RunCmdOnNode executes a command from within the given ip node
+func RunCmdOnNode(cmd string, serverIP string) (string, error) {
+	host := serverIP + ":22"
+	conn := configureSSH(host)
+
+	stdout, stderr, err := runsshCommand(cmd, conn)
+	if err != nil {
+		return fmt.Errorf(
+			"Command: %s \n failed on Node: %s with error: %w",
```

Comment:
Std practice not to capitizalize error messages:
```suggestion
			"command: %s \n failed on Node: %s with error: %w",
```